### DISCUSSION TITLE
Remind me, how does the site send email (for forgotten passwords etc)?

### DISCUSSION THREAD

**Discussion by ClearSky0** [ryanhowdy/fcms#640] [Q&A] [answered]
Hi there

In anticipation of the new version ;-), I'm trying to revive my old FCMS site that has lay dormant for about ten years.  It's also part of a side-project to get a Raspberry Pi set-up with a few interesting things.

Can anyone remind me how the email works for notifications and password resets please?  I mean in terms of what the requirements are on the server?  I'm trying to use Nginx now, but used Apache last time.

Thanks in advance, Gary.

**Comment by ryanhowdy** [comment, marked as answer]
It uses sendmail.

**Reply by ClearSky0** [reply]
Thank you. I've got it working now after configuring msmtp to connect to my zoho account.
